test/resources/ostree.resource

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation    Keywords for OSTree-based systems
+
+Library     SSHLibrary
+
+
+*** Variables ***
+${BACKUP_STORAGE}       /var/lib/microshift-backups
+${HEALTH_FILE}          /var/lib/microshift-backups/health.json
+
+
+*** Keywords ***
+Get Booted Deployment Status
+    [Documentation]    Get rpm-ostree status of booted deployment as a json
+
+    ${ostree_status}    ${rc}=    Execute Command
+    ...    rpm-ostree status --booted --json
+    ...    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+    ${json}=    Evaluate    json.loads('''${ostree_status}''')    json
+    RETURN    ${json}
+
+Get Booted Deployment ID
+    [Documentation]    Get ID of currently booted deployment
+
+    ${status}=    Get Booted Deployment Status
+    RETURN    ${status["deployments"][0]["id"]}
+
+Get Booted Deployment Backup Prefix Path
+    [Documentation]    Get backup path prefix for current deployment
+    ...
+    ...    Prefix path is BACKUP_STORAGE/deployment-id.
+    ...    Globbing directories starting with the prefix will yield
+    ...    list of backups for the deployment.
+
+    ${current_deployment_id}=    Get Booted Deployment ID
+    Should Not Be Empty    ${current_deployment_id}
+    Should Not Be Empty    ${BACKUP_STORAGE}
+    ${backup_path}=    Catenate    SEPARATOR=/    ${BACKUP_STORAGE}    ${current_deployment_id}
+    RETURN    ${backup_path}
+
+Get System Health
+    [Documentation]    Get system health information from health.json file
+
+    ${health}    ${stderr}    ${rc}=    Execute Command
+    ...    jq -r '.health' ${BACKUP_STORAGE}/health.json
```

Comment:
'Evaluate    json.loads' was used above in keyword "Get Booted Deployment Status". Should we be consistent in the pattern we use to ingest and parse json?

Reply:
As a side note, 'jq' is used by scripts such as microshift_set_healthy.sh, so it's not a bad thing that we are also using it in our test scripts. My initial comment was just a reaction to seeing json parsed in the same PR using two different approaches.

Reply:
That's a good point. `jq` here felt simpler, otherwise I'd have either `cat` and use `json.loads` or... I don't know what, I don't think `SSHLibrary.Get File` supports privilege escalation and `health.json` is only readable by root

Reply:
I like the idea of keywords for grabbing files only readable by root. [Save Default MicroShift Config](https://github.com/openshift/microshift/blob/main/test/resources/microshift-config.resource#L13) uses `cat` to grab the contents of the config file, and it could use the same reusable keyword for that step. Then something like [Upload String To File](https://github.com/openshift/microshift/blob/main/test/resources/microshift-config.resource#L13) could be modified to take arguments for the owner and mode of the output file.